examples/pub-mediadevice/main.go

```diff
@@ -112,15 +111,25 @@ func main() {
 	}
 	vp8Params.BitRate = 100000 // 100kbps
 
+	opusParams, err := opus.NewParams()
+	if err != nil {
+		panic(err)
+	}
+	opusParams.BitRate = 32000 // 32kbps
+
 	fmt.Println(md.EnumerateDevices())
 
 	s, err := md.GetUserMedia(mediadevices.MediaStreamConstraints{
-		Video: func(c *mediadevices.MediaTrackConstraints) {
-			c.FrameFormat = prop.FrameFormat(frame.FormatYUYV)
+		// Video: func(c *mediadevices.MediaTrackConstraints) {
```

Comment:
Did you mean to commit these changes since this conflicts with the frontend logic of the `pub-mediadevices`?